ladybug_pandas/extension_types/dtype.py

```diff
@@ -115,7 +115,7 @@ def to_header(self):
 
     @property
     def type(self):
-        return np.float
+        return float
```

Comment:
Hi, @devngc -

Can we make this change backward compatible?

```python
try:
    return np.float
except AttribueError:
    return float
```

Reply:
returning a float type should be backward compatible since its a python data type.

Reply:
I pushed what you suggested